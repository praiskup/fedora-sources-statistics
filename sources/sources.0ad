SHA512 (0ad-0.0.26-alpha-unix-build.tar.xz) = aaf647d5d8454c244015d2a198beeaaebc571a5bc96446f3acff8dbd05f9843029c500bf4162651a5e1fcdb42bd5fb5b4f5c512c78372479fbd8565dd093f272
SHA512 (0001-Bug-1654457-Update-virtualenv-to-20.0.31.-r-mhentges.patch) = 0b9107cc6d5abc57f1a63dec435e1e5e836caeb0a27e32e77c2d60f67fd598c195e93540139ebf1a9bbaabf84b9ece2986282c829a7c056384e54e2dece2532d
