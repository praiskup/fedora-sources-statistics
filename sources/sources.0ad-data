SHA512 (0ad-0.0.26-alpha-unix-data.tar.xz) = 8fe29fce68737d5b7001c8c2a7198766583b22556129cd2d927823da9b2c58f224fb1207f35bd03abea9380cf24d00ab6257b0dba86ce3ad1f357d7006f051d6
