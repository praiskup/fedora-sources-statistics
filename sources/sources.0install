SHA512 (0install-2.18.tbz) = 6d4734754951fad9caad5d876a1301e283bdd77a080d6601d57b21c540a30a616b18f664cbd4f86748c90de41cf3ef54f3a883fd75f638abbe3630a0320e4a7f
SHA512 (0install-2.18.tbz.sig) = bb323a22f533daf3c561439f90fe29f42aae11b10c2f10c89ce6f8819e0d26139430167daa612dd6f8eb2854163c8596f5dba202f4c464fada3a686da2677a56
