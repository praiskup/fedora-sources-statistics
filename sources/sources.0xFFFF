SHA512 (0xFFFF-0.10.tar.gz) = 3eaa86ea3a56faedd9137f777a0a4deb13655f4689db02a22b489da7c040b45a129d8237268e7c70cd56d333d7a13257368c8c06ddbfa0bdaad4bb9bd4ab5b36
