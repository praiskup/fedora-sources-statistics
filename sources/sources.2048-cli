e940e430d4222718dd04ad88bd0daf2a  2048-cli-0.9.1.tar.gz
