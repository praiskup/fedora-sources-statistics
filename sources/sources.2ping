SHA512 (2ping-4.5.1.tar.gz) = 13ab8c34d6b4e8d53bc0f1c33afa1c71bf61313d29350d2211a44557b244062cb090cf5d66cc6eed58f9f4c41a667bb24a8c1155243864e8343c741770b60cd6
