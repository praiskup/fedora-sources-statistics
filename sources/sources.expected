SHA512 (expected-1.1.0.tar.gz) = ce970c31582869af9d0b3349f386db207dd4881db0bdfd3744331b0a62fe2886dde598a75882fb00254afa3549fb9d4c2bd1ff7682744891d403edfd4ff73492
